File: src/models/feature.ts

export class _Feature {
    constructor(
        public module: string,
        public functionality: string,
        public tag: string,
        public discontinued: boolean,
        public fragment: string,
        public included: boolean,
        public notYetAvailable: boolean,
        public fromPackage: boolean,
        public standard: string,
        public professional: string,
        public premium: string,
        public enterprise: string,
        public customer: boolean,
        public allowISO: string,
        public denyISO: string
    ) {}
}

File: Models/Feature.cs
 namespace Configurator.Models
 {
    public class Feature
    {
        public string Module { get; set; }
        public string Functionality { get; set; }
        public string Tag {get; set;}
        public bool Discontinued {get; set;}
        public string Fragment {get; set;}
        public string AllowISO {get; set;}
        public string DenyISO {get; set;}
        public string Standard {get; set;}
        public string Premium {get; set;}
        public string Professional {get; set;}
        public string Enterprise {get; set;}
        public bool Included {get; set;}
        public bool NotYetAvailable {get; set;}
    }
 }
 
